pkg/addons/addons.go

```diff
@@ -323,10 +329,14 @@ func Start(profile string, toEnable map[string]bool, additional []string) {
 
 	out.T(out.AddonEnable, "Enabling addons: {{.addons}}", out.V{"addons": strings.Join(toEnableList, ", ")})
 	for _, a := range toEnableList {
-		err := Set(a, "true", profile)
-		if err != nil {
-			// Intentionally non-fatal
-			out.WarningT("Enabling '{{.name}}' returned an error: {{.error}}", out.V{"name": a, "error": err})
-		}
+		go func() {
```

Comment:
I think go routines in for loops usually require something like:

```go
go func(a string) {
  /// code
}(a)
```
since the value of a will change

[ref](https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables)